docs/_sass/_landingPage.scss

```diff
@@ -715,15 +715,21 @@ body {
 
 // HAMBURGER MENU
 .header-section__hamburger {
-  min-width: 25px;
+  min-width: 15px;
   -moz-transition: all 0.3s ease;
   transition: all 0.3s ease;
 }
 
+
 .header-section__sap-icon--menu2::before {
-  font-size: 20px;
   transition: opacity 0.3s, transform 0.3s;
   color: $home-page-action-color;
+  @media only screen and (max-width: 1024px) {
+   font-size: 14px;
+  }
+  @media only screen and (min-width: 1024px) {
+    font-size: 20px;
+  }
```

Comment:
could you remove `max-width: 1024px` query? It's seems to be quite useless. You can put `font-size: 14px` instead of whole query.